recipes-devtools/ruby-cross_3.0.0.bb

```diff
@@ -37,17 +36,12 @@ SHRT_VER = "${@oe.utils.trim_version("${PV}", 2)}"
 
 SRC_URI_append = " \
                   http://cache.ruby-lang.org/pub/ruby/${SHRT_VER}/ruby-${PV}.tar.gz \
-                  file://0001-extmk-fix-cross-compilation-of-external-gems.patch \
-                  file://0002-Obey-LDFLAGS-for-the-link-of-libruby.patch \
                   file://remove_has_include_macros.patch \
-                  file://0001-Modify-shebang-of-libexec-y2racc-and-libexec-racc2y.patch \
                   file://0001-template-Makefile.in-do-not-write-host-cross-cc-item.patch \
                   file://0001-Makefile-cross-compile-fixes.patch \
-                  file://autoconf270.patch \
```

Comment:
potential issue: Upstream-Status in '0001-template-Makefile.in-do-not-write-host-cross-cc-item.patch' doesn't pick from valid classifiers Accepted,Backport,Denied,Inappropriate,Pending,Submitted